src/virtualize.js

```diff
@@ -132,7 +132,16 @@ export default function virtualize(MyComponent) {
       }
 
       this.setState(nextState, () => {
-        this.setWindowTimer();
+        // We are going backward, but reached first rendered item, let's render previous right away, with no timer
+        if (indexDiff < 0 && index > 0 && indexContainer === 0) {
```

Comment:
index > 0 checks if there are more slides to display to the left. Maybe that's unnecessary, but I'm double checking.